packages/connect-core/src/transactions/TransactionRequest.ts

```diff
@@ -22,6 +29,8 @@ export default class TransactionRequest {
     from: string
     data: string
   }
+  readonly description?: string
```

Comment:
I decided to expand the `TransactionRequest` object to include both the `description` and `annotatedDescription` values. What do you think? I wonder if this might introduce problems when interacting with other libraries and we should handle it as a separate `TransactionDescription` object.

Reply:
No, I just wonder if having these extra parameters could provoke problems when sending the transaction through Ethers or Web3.